### PR TITLE
Add Polkadot

### PR DESCRIPTION
**Pull request by ruiparitydata**
**Description**: added Polkadot chain

**Contribution Details**:
Used the [Subscan API](https://support.subscan.io/api-4193060) to get the list of active validators

### CODE REVIEW

File: core/chains/chain.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ const (
 	AVAX  Token = "AVAX"
 	BLD   Token = "BLD"
 	BNB   Token = "BNB"
+	DOT   Token = "DOT"
```
Comment (xenowits) [inline core/chains/chain.go:27]: Please also add polkadot to the list of supported chains

File: core/chains/polkadot.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	}
+
+	// Loop through the validators bonded amounts
+	for _, ele := range response.Data.List {
+		bondedTotal, err := strconv.ParseInt(ele.BondedTotal, 10, 64)
+		if err != nil {
+			log.Println(err)
+			continue
+		}
+		votingPowers = append(votingPowers, bondedTotal)
```
Comment (xenowits) [inline core/chains/polkadot.go:68]: add a newline before this statement

**Comment by xenowits** [comment]
Hey @ruiparitydata , can you please share the NC value for polkadot that your code calculates? A screenshot would be nice

**Comment by ruiparitydata** [comment]
Thanks for reviewing @xenowits, please let me know if I added the newline in the correct place. The chain was added to the readme file as a supported chain.

Screenshot with the NC value:
Screenshot 2023-12-12 at 18 52 50

**Review by xenowits — APPROVED** [review]

**Pull request merged**